### PR TITLE
vendor: bump etcd v3.4.9

### PR DESCRIPTION
**Pull request by hexfusion**
bumping etcd client to v3.4.9 to match server https://github.com/openshift/etcd/pull/52

Signed-off-by: Sam Batschelet <sbatsche@redhat.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -15,7 +15,7 @@ require (
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
 	github.com/vishvananda/netlink v1.0.0
-	go.etcd.io/etcd v0.0.0-20200401174654-e694b7bb0875
+	go.etcd.io/etcd v0.5.0-alpha.5.0.20200520232829-54ba9589114f
```
Comment (retroflexer) [inline go.mod:18]: Why does it say 5.0-alpha?

Comment (hexfusion) [reply go.mod:18]: its just the base tag for the etcd repo it gets pulled in by `go mod` automation. the sha is what is important  `54ba9589114f`

Comment (hexfusion) [reply go.mod:18]: additional reference https://github.com/kubernetes/kubernetes/pull/92075/files#diff-37aff102a57d3d7b797f152915a6dc16R102

Comment (retroflexer) [reply go.mod:18]: OK. Thanks.

Comment (retroflexer) [reply go.mod:18]: Yeah. Here is the top commit in 3.4.9 in etcd (54ba9589114fc3fa5cc36c313550b3c0c0938c91):

https://github.com/etcd-io/etcd/commit/54ba9589114fc3fa5cc36c313550b3c0c0938c91

**Pull request merged**